bdk-python/scripts/generate-linux.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+${PYBIN}/python --version
+${PYBIN}/pip install -r requirements.txt
+
+echo "Generating bdk.py..."
+cd ../bdk-ffi/
+cargo run --bin uniffi-bindgen generate src/bdk.udl --language python --out-dir ../bdk-python/src/bdkpython/ --no-format
+
+echo "Generating native binaries..."
+rustup default 1.67.0
```

Comment:
In the future this should probably come from en env variable set by the github action, but this is fine until we need to update it to a new version.